ProjectLighthouse/Database.cs

```diff
@@ -285,6 +285,13 @@ public async Task UnqueueLevel(User user, Slot queuedSlot)
 
     #region Game Token Shenanigans
 
+    public async Task<string> UsernameFromGameToken(GameToken? token)
+    {
+        if (token == null) return "";
```

Comment:
This is a strange way to handle this, I would throw an exception here.

Is there a reason we want to return an empty string?